snapcraft.yaml

```diff
@@ -110,6 +110,7 @@ parts:
       # This list includes the dependencies for curtin and probert as well,
       # there doesn't seem to be any real benefit to listing them separately.
       - cloud-init
+      - dctrl-tools
```

Comment:
I guess we could just stage /usr/bin/grep-dctrl but the whole package is fairly tiny.

Reply:
Yes - and no Depends so I guess I'll keep the whole package.